http-service/src/main/java/com/laiteam/echowall/httpservice/response/FollowResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.laiteam.echowall.httpservice.response;
+
+import com.laiteam.echowall.dal.entity.Profile;
+import com.laiteam.echowall.httpservice.utils.BeanUtils;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Setter
+@Getter
+public class FollowResponse {
```

Comment:
I don't think you have to create a new FollowResponse using ProfileResponse should be enough